opentok/opentok-meet
# [OPENTOK-44700] Adding Anvil endpoint

#### What is this PR doing?
Adding anvil endpoint to meet for webview programmable renderer.

#### How should this be manually tested?
Run meet:
- Click on Webcomposer
- Check a message should appear with this content: Starting WebView Composer... 
- Click on the url created. After around 20 secs the rider should have been started.
- Check you can see the sub for the publisher you have in the regular session plus a watermark and stats deployed.
- Go back to the regular session, and click again on Webcomposer to stop the rider.
- Now you shouldn't see the sub in the rider session.

#### What are the relevant tickets?
OPENTOK-44700
